cvat/apps/engine/serializers.py

```diff
@@ -2432,7 +2480,24 @@ def create(self, validated_data):
 
     # pylint: disable=no-self-use
     @transaction.atomic
-    def update(self, instance, validated_data):
+    def update(self, instance: models.Task, validated_data: dict):
+        organization_id = validated_data.get("organization_id")
+        organization_slug = None
+
+        workspace_transferring = (
+            "organization_id" in validated_data
+            and organization_id != instance.organization_id
+        )
+
+        if (
+            workspace_transferring and (set(validated_data.keys()) - {
+                'source_storage', 'target_storage', 'organization_id'
+            })
+        ):
+            raise serializers.ValidationError(
+                "Task attributes cannot be updated during transferring to another workspace"
+            )
+
```

Comment:
Given that most of the code below is irrelevant if a transfer is requested, can we extract this logic into a new function or at least make a big if-else in this one?